README.md

```diff
@@ -1188,6 +1191,43 @@ All options from BDD tests do also apply here.
 
 **NOTE:** Performance tests should be run without concurrency.
 
+#### List of test tags
+
+| Tag name           | Tag meaning                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------- |
+| @smoke             | Smoke tests verifying basic functionality                                          |
+| @performance       | Performance tests                                                                  |
+| @olm               | OLM integration tests                                                              |
+| @travelagency      | Travel agency tests                                                                |
+|                    |                                                                                    |
+| @disabled          | Disabled tests, usually with comment describing reasons                            |
+| @cli               | Tests to be executed only using cli                                                |
```

Comment:
@cli is the Kogito CLI? The another way is to use directly the OC command, right?

Reply:
It marks the scenarios which require Kogito CLI (for example they test that Kogito CLI installs the Kogito operator when new project is created).